sessionctx/variable/tidb_vars.go

```diff
@@ -333,6 +333,12 @@ const (
 
 	// TiDBUsePlanBaselines indicates whether the use of plan baselines is enabled.
 	TiDBUsePlanBaselines = "tidb_use_plan_baselines"
+
+	// TiDBIsolationReadLabels indicates the tidb only read from the stores having at least one label in IsolationReadLabels.
+	TiDBIsolationReadLabels = "tidb_isolation_read_labels"
```

Comment:
still quite confused, why we need to introduce this session variable?

Reply:
You can check the test for more information.